src/services/index.js

```diff
@@ -1,7 +1,9 @@
 import sendMail from './sendMail';
 import findUser from './findUser';
+import novel from './novel';
 
 export default {
   sendMail,
-  findUser
+  findUser,
+  novel
```

Comment:
I feel you should rename the service novel, the naming seems ambiguous

Reply:
@allebd can you provide context to why you think it's ambiguous?

Reply:
Reviewed and changed